awslabs/aws-lambda-powertools-typescript
# feat: Add codespaces/gitpod support

## Description of your changes

- Add configuration for codespaces and gitpod
- Set node version via `.nvmrc` to latest v14
- Set npm version to `npm@next-8`

### How to verify this change

- Create new codespaces workspace from PR
- Open PR in [GitPod](https://gitpod.io/#https://github.com/gyft/aws-lambda-powertools-typescript/tree/feat-codespaces)
- And run `npm test` or pushing a commit to check if pre-commits work.

### Related issues, RFCs

- #115
- #426

### PR status

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

## Checklist



---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
